common/components/common/volunteers/VolunteerCard.jsx

```diff
@@ -62,21 +66,21 @@ class VolunteerCard extends React.PureComponent<Props> {
   _renderApplicationMenuLinks(): ?Array<React$Node>  {
     if (this.props.volunteer && this.props.volunteer.isCoOwner) {
       return [
-          (<MenuItem className="VolunteerCard-caution" onSelect={() => this.props.onDemotionButton(this.props.volunteer)} key="0">Demote</MenuItem>),
-          (<MenuItem className="VolunteerCard-danger" onSelect={() => this.props.onDismissButton(this.props.volunteer)} key="1">Remove</MenuItem>)
+          (<MenuItem onSelect={() => this.props.onDemotionButton(this.props.volunteer)} key="0"><i className={`${GlyphStyles.Pushpin} fa-fw`}></i> Demote</MenuItem>),
```

Comment:
We should add functionality for adding 'fa-fw' to glyph classes, similar to how we accommodate glyph sizes.  Add a TODO if you don't feel like doing it this time around.